Agendas/ST-vesting.md

```diff
@@ -0,0 +1,73 @@
+# TABLE OF CONTENTS
+1 [Security Target Introduction](#1-security-targets-introduction)<br>
+1.1 Security Target Reference <br>
+1.2 TOE Reference <br>
+1.3 Terminology and Acronyms<br>
+1.4 Product Overview<br>
+1.5 TOE Overview <br>
+1.6 TOE Description<br>
+2 Conformance Claims<br>
+3 TOE Security Problem Definition<br>
+3.1 Assumption<br>
+3.2 Threats<br>
+3.3 Organizational Security Policies<br>
+4 Security Objectives<br>
+4.1 Security Objectives for the TOE<br>
+4.2 Security Objectives for the Operational Environment<br>
+5 Extended Components<br>
+5.1 Extended Security Functional Requirement (SFR)<br>
+5.2 Extended Security Assurance Requirement (SAR)<br>
+6 TOE Security Requirements<br>
+6.1 Conventions<br>
+6.2 Security Functional Requirements (SFR)<br>
+6.3 Security Assurance Requirements<br>
+7 TOE Summary Specifications<br>
+7.1 User Data Protection<br>
+7.2 Identification and Authentication<br>
+7.3 Security Management<br>
+7.4 TOE Access<br>
+8 Rationale<br>
+8.1 Protection Profile Conformance Claim Rationale<br>
+8.2 Security Objectives Rationale<br>
+8.3 Extended Security Functional Requirement Rationale<br>
+8.4 Extended Security Assurance Requirement Rationale<br>
+8.5 Security Functional Requirements Rationale<br>
+
+# 1. Security Targets Introduction
+
+1.1 Security Target Reference <br>
+
+TODO
+
+1.2 TOE Reference <br>
+
+TODO
+
+1.3 Terminology and Acronyms<br>
+
+TODO
+
+1.4 Product Overview<br>
```

Comment:
It's usually a good place to have a figure that explains the different components of the whole product and the things it interacts with. This allows then to highlight which parts are in the TOE and which parts are outside :)

Reply:
The figure

https://github.com/aleeusgr/potential-robot/issues/134